org-formation/600-access/organization-tasks.yaml

```diff
@@ -0,0 +1,14 @@
+
+# A service account for https://github.com/nlpsandbox/aws-cloudformation
+NlpSandboxServiceAccount:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/service-account.yaml
+  StackName: NlpSandboxServiceAccount
+  MaxConcurrentStacks: 10
+  Parameters:
+    ServiceUserPolicies:
+      - arn:aws:iam::aws:policy/AdministratorAccess
```

Comment:
Does this account really need this policy? If its in sandbox, it doesn't matter much, but one day it will migrate to a prod account, so we need to get the permissions right in the first place.

Reply:
We don't need to deploy this same policy for prod account.  We can create a new `NlpProdServiceAccount` with a different set of policies and deploy that to prod.

Reply:
+1

Reply:
That sounds right... except that we won't know what the correct perms should be unless the sandbox prototype proves it

Reply:
@thomasyu888 will figure that out in sandbox then we can tighten it up before deploying to scicomp

Reply:
I agree, sandbox should be where we figure out the problems, including what permissions are needed to run...
Is this the right place to define the policy (i.e. should we not just have to setup a service account that assumes
a role, the role itself being defined by the app dev)?

Reply:
I think this PR is doing exactly what you are suggesting @xschildw?   I'm not sure what you feel needs to change about it.

Reply:
My bad, you're correct. My only issue is with not specifying the actual permissions in sandbox.

Reply:
It seems like what i need is `arn:aws:iam::aws:policy/AmazonEC2FullAccess`.  This user needs to: 

- create, update, delete ec2
- create, update, delete ELB2 resources (listeners, load balancer, Target group)
- create, update, delete security groups

Reply:
That seems to right to me... you can either try it with the permission set you think it needs and then confirm it works, or give it the admin policy and guess later... the second method seems wrong, but since this is sandbox it doesn't matter much